### ISSUE TITLE
System.ArgumentException: parsing

### ISSUE THREAD

**Issue by johnsimons** [shaneray/ShaneSpace.VisualStudio.InvisibleCharacterVisualizer#3]
Installed in VS 2017, started VS and got the following error in `ActivityLog.xml`:

```
System.ArgumentException: parsing &quot;[\0-\x08\u000A-\u001F\x7F-\x9F\U00A0\xAD\u0378\u0379\u037F-\u0383\u038B\u038D\u03A2\u0528-\u0530\u0557\u0558\u0560\u0588\u058B-\u058E\u0590\u05C8-\u05CF\u05EB-\u05EF\u05F5-\u0605\u061C\u061D\u06DD\u070E\u070F\u074B\u074C\u07B2-\u07BF\u07FB-\u07FF\u082E\u082F\u083F\u085C\u085D\u085F-\u089F\u08A1\u08AD-\u08E3\u08FF\u0978\u0980\u0984\u098D\u098E\u0991\u0992\u09A9\u09B1\u09B3-\u09B5\u09BA\u09BB\u09C5\u09C6\u09C9\u09CA\u09CF-\u09D6\u09D8-\u09DB\u09DE\u09E4\u09E5\u09FC-\u0A00\u0A04\u0A0B-\u0A0E\u0A11\u0A12\u0A29\u0A31\u0A34\u0A37\u0A3A\u0A3B\u0A3D\u0A43-\u0A46\u0A49\u0A4A\u0A4E-\u0A50\u0A52-\u0A58\u0A5D\u0A5F-\u0A65\u0A76-\u0A80\u0A84\u0A8E\u0A92\u0AA9\u0AB1\u0AB4\u0ABA\u0ABB\u0AC6\u0ACA\u0ACE\u0ACF\u0AD1-\u0ADF\u0AE4\u0AE5\u0AF2-\u0B00\u0B04\u0B0D\u0B0E\u0B11\u0B12\u0B29\u0B31\u0B34\u0B3A\u0B3B\u0B45\u0B46\u0B49\u0B4A\u0B4E-\u0B55\u0B58-\u0B5B\u0B5E\u0B64\u0B65\u0B78-\u0B81\u0B84\u0B8B-\u0B8D\u0B91\u0B96-\u0B98\u0B9B\u0B9D\u0BA0-\u0BA2\u0BA5-\u0BA7\u0BAB-\u0BAD\u0BBA-\u0BBD\u0BC3-\u0BC5\u0BC9\u0BCE\u0BCF\u0BD1-...&quot; - Unrecognized escape sequence \U.&#x000D;&#x000A;   at System.Text.RegularExpressions.RegexParser.ScanCharEscape()&#x000D;&#x000A;
   at System.Text.RegularExpressions.RegexParser.ScanCharClass(Boolean caseInsensitive, Boolean scanOnly)&#x000D;&#x000A;
   at System.Text.RegularExpressions.RegexParser.CountCaptures()&#x000D;&#x000A;   at System.Text.RegularExpressions.RegexParser.Parse(String re, RegexOptions op)&#x000D;&#x000A;   at System.Text.RegularExpressions.Regex..ctor(String pattern, RegexOptions options, TimeSpan matchTimeout, Boolean useCache)&#x000D;&#x000A;
   at System.Text.RegularExpressions.Regex..ctor(String pattern, RegexOptions options)&#x000D;&#x000A;
   at ShaneSpace.VisualStudio.InvisibleCharacterVisualizer.InvisibleCharacterTagger..ctor(ITextBuffer buffer)&#x000D;&#x000A;
   at ShaneSpace.VisualStudio.InvisibleCharacterVisualizer.InvisibleCharacterTaggerProvider.&lt;&gt;c__DisplayClass0_0`1.&lt;CreateTagger&gt;b__0()&#x000D;&#x000A;
   at Microsoft.VisualStudio.Utilities.PropertyCollection.GetOrCreateSingletonProperty[T](Object key, Func`1 creator)&#x000D;&#x000A;
   at ShaneSpace.VisualStudio.InvisibleCharacterVisualizer.InvisibleCharacterTaggerProvider.CreateTagger[T](ITextBuffer buffer)&#x000D;&#x000A;
   at Microsoft.VisualStudio.Text.Tagging.Implementation.TagAggregator`1.GatherTaggers(ITextBuffer textBuffer)
```

**Comment by shaneray** [comment]
Thanks for the report.  Will check it out ASAP.

**Comment by shaneray** [comment]
I have found the issue and will be pushing an update later tonight or tomorrow.  Thanks again for the report.

**Comment by shaneray** [comment]
Fixed by 1.3 release.
